docs/integrations/keras.rst

```diff
@@ -1,112 +1,319 @@
+
+.. code-block:: bash
+
+   pip install psutil
+
+To see your hardware metrics, you can click on the project |link| generated by Neptune above, and then select logs. You should be presented with a realtime dashboard as shown below:
+
+.. image:: ../_static/images/integrations/keras-integration/monitoring.png
+
+
+
+**Step 6**
+In a new cell of your notebook, import all the necessary packages needed for this project.
+
+.. code-block:: python3
+
+   ...
+   ...
+   from neptunecontrib.monitoring.keras import NeptuneMonitor
+
+Notice that we're importing the NeptuneMonitor from the neptunecontrib package? This is because we are going to be integrating neptune with keras. Neptune can also be integrated with other libraries easily. You can find some useful link on how to achieve this below.
+
+.. **TODO: SOME SEO LINKS HERE**
+
+
+Now since this is a text classification dataset, there are numerous preprocessing steps you will follow in order to get your data into numeric form which you can pass to your model. We assume you have performed these steps and have the saved some important parameters like vocabulary size, input size and so on. 
+In code block below, we create a dictionary to hold these parameters, as it will be used to initialize a new neptune training experiment.
+
+.. code-block:: python3
+
+   # parameters
+   PARAMS = {'vocab_size':vocab_size,
+            'max_length': max_length,
+            'epochs': 100,
+            'batch_size': 64,
+            'input_size': 24,
+            'metric': 'accuracy',
+            'loss': 'binary_crossentropy',
+            'optimizer': 'rmsprop',
+            'dropout': 0.5}
+
+
+   #create first model experiment
+   neptune.create_experiment(name='training_model_exp1', params=PARAMS)
+   
+.. image:: ../_static/images/integrations/keras-integration/out2.png
+
+
+Notice that the **PARAMS** dictionary contains mostly model parameters like input size, epochs, metrics and so on. After parameter initialization, you'll create your keras model.
+
+.. code-block:: python3
+
+   # define the model
+   model = Sequential()
+   model.add(Embedding(vocab_size, PARAMS['input_size'], input_length=max_length))
+   model.add(Flatten())
+   model.add(Dense(500, activation='relu'))
+   model.add(Dense(200, activation='relu'))
+   model.add(Dropout(PARAMS['dropout']))
+   model.add(Dense(100, activation='relu'))
+   model.add(Dense(1, activation='sigmoid'))
+
+   # compile the model
+   model.compile(optimizer=PARAMS['optimizer'], loss=PARAMS['loss'], metrics=[PARAMS['metric']])
+
+   # summarize the model
+   print(model.summary())
+
+
+.. image:: ../_static/images/integrations/keras-integration/model-summary.png
+
+
+
+The model is pretty basic, and uses an |embedding| layer as the input because you're working with text inputs of large dimensions. The output layer is a sigmoid node, because this is a binary classification problem (Spam or Not-Spam).
```

Comment:
This doesn't add value to the reader context. 
They want to read about keras integration -> again pls remember it is a docs page not a blog. 
You can take a look at optune/lightning